index.js

```diff
@@ -42,4 +43,8 @@ const Pulsar = {
   LogLevel,
 };
 
+(() => {
```

Comment:
It is better that move the generate cert to new file.

Reply:
I want to use `index.js` as some preparation after installation, and if there are other requirements, can add it to this method.

> and this file is pure javascript without native module.
 Generating certificates is part of the node client

Reply:
But there's no point in generating it too many times.

Reply:
oh, I see. I initially thought this method would only be called once. It turns out that whenever the user `require index.js` it will be executed